templates/search.html

```diff
@@ -0,0 +1,19 @@
+{% if 'search' in DIRECT_TEMPLATES and USE_TIPUE_SEARCH %} {% extends 'base.html' %} {% block title %} Search - {{ SITENAME|striptags }} {% endblock title %} {% block content %}
```

Comment:
why this if here? also Search is not translated.

Reply:
```suggestion
{% USE_TIPUE_SEARCH %} {% extends 'base.html' %} {% block title %} Search - {{ SITENAME|striptags }} {% endblock title %} {% block content %}
```
Modified to remove "if 'search' in DIRECT_TEMPLATES", I do not understand what you mean by translated nor how to implement it, would you mind showing me where this is implemented elsewhere please?